.buildkite/commands/run-unit-tests.sh

```diff
@@ -17,6 +17,16 @@ elif [ "$1" == "image-editor" ]; then
     test_results_dir="libs/image-editor/build/test-results"
     test_log_dir="${test_results_dir}/testReleaseUnitTest/*.xml"
     code_coverage_report="libs/image-editor/build/reports/kover/reportRelease.xml"
+elif [ "$1" == "fluxc" ]; then
+    test_suite=":fluxc:testReleaseUnitTest :fluxc:koverXmlReportRelease"
+    test_results_dir="fluxc/build/test-results"
+    test_log_dir="${test_results_dir}/testReleaseUnitTest/*.xml"
+    code_coverage_report="fluxc/build/reports/kover/reportRelease.xml"
+elif [ "$1" == "WordPressLoginFlow" ]; then
+    test_suite=":WordPressLoginFlow:testReleaseUnitTest :WordPressLoginFlow:koverXmlReportRelease"
+    test_results_dir="WordPressLoginFlow/build/test-results"
+    test_log_dir="${test_results_dir}/testReleaseUnitTest/*.xml"
+    code_coverage_report="WordPressLoginFlow/build/reports/kover/reportRelease.xml"
```

Comment:
Yea, I am not sure as well, but maybe yes, especially if it is cheat, just to keep the configuration consistent on each module, and then to not have to worry about missing that going forward, say we start adding more unit tests for `Login`... 🤷 

PS: Totally unrelated @wzieba , organizational wise, now that we have this chance and just to keep things consistent and avoid confusion/questions down the line:
- Why not moving all these vendored modules into their own folder, even maybe calling it `vendored`. Just like it happens with the `libs` related modules. Actually, I would be totally okay if we move them to this existing `libs` folder, since some of those libs within that folder went through this exact process, them being vendored and all. 🤔 
- Why not renaming the `WordPressLoginFlow` module to plain `login`. 🤔

Reply:
> Why not moving all these vendored modules into their own folder, even maybe calling it vendored.

I tried to reduce git history manipulation to minimum. I also don't see benefits of introducing these modules into a directory: for me, it makes task names longer, instead of `fluxc:assembleRelease` we'd have to run `:libs:fluxc:assembleRelease`. What benefits do you see that moving these 2 modules to a directory would bring?

> Why not renaming the WordPressLoginFlow module to plain login. 🤔

We can rename the project name to `login` without renaming the directory. WDYT?

Reply:
> I tried to reduce git history manipulation to minimum.

👍 

> I also don't see benefits of introducing these modules into a directory: for me, it makes task names longer, instead of fluxc:assembleRelease we'd have to run :libs:fluxc:assembleRelease.

Yea, true, but then again, we rarely run there `assemble` commands on the lib module itself and usually run the main app module assemble command for all, so I don't think this is a problem, having the names longer. 🤷 

> What benefits do you see that moving these 2 modules to a directory would bring?

Not only these 2 modules, but actually all 4 of them, if I recall correctly:
- WordPressLoginFlow
- fluxc-annotations
- fluxc-processor
- fluxc

The benefits I see is mostly organizational wise (as I said earlier), just so that we don't pollute the root level directories with extra (temporar) modules/folders... 🤷 

I also think that naming that folder as `vendored` would also help others understand where and why those modules got moved into the repo. Else, using the existing `libs` folder is perfectly fine as well, and one less folder to worry about.

> We can rename the project name to login without renaming the directory. WDYT?

Hmmm, I don't know, wouldn't that further complicate things, having 2 names for the same things, a project and a folder name. 🤔 I think it is better to stick with `WordPressLoginFlow`, that is, if renaming that to `login` is too troublesome.

Reply:
I see @ParaskP7 👍 What's your preference then: moving these modules to `libs` or creating a new directory: `vendored`? It's worth to note that we already have `vendor` directory created when ruby dependencies are installed

Reply:
`libs` it is then @wzieba , that would have been my preferred approach no matter if the `vendor` directory existed or not, especially since some of the other existing `libs` related modules were vendored in there too. I only suggested `vendored` in case we wanted to separate these two directories, just because `fluxc` is to be replaced at some point with `rs`. Thanks for asking! 🙏

Reply:
Thanks 👍 . Vendored projects are now in `libs` with `WordPressLoginFlow` renamed to simply `login` 🙌